utils/time.js

```diff
@@ -0,0 +1,22 @@
+export function minutesFromMs(ms, mode = "round") {
+  const v = ms / 60000;
+  if (mode === "floor") return Math.floor(v);
+  if (mode === "ceil") return Math.ceil(v);
+  return Math.round(v);
+}
+
+export function formatMinutesHm(totalMinutes) {
+  const h = Math.floor(totalMinutes / 60);
+  const m = totalMinutes % 60;
+  return h > 0 ? `${h}h ${m}m` : `${m}m`;
+}
+
+export function formatDurationMs(ms, opts = {}) {
+  return formatMinutesHm(minutesFromMs(ms, opts.mode ?? "round"));
+}
+
+// Discord timestamp styles: 't','T','d','D','f','F','R'
+export function discordTs(input, style = "t") {
```

Comment:
Could we rename this to something more explicit, like `formatDiscordTimestamp` or `generateDiscordTimestamp`, to make it clearer what it returns?

Might also be worth adding a short comment with the available Discord timestamp formats for quick reference while building:
Code | Example | Type
-- | -- | --
t | 16:20 | short time
T | 16:20:30 | long time
d | 20/04/2025 | short date
D | 20 April 2025 | long date
f | 20 April 2025 16:20 | short date/time
F | Sunday, 20 April 2025 16:20 | long date/time (full)
R | 2 hours ago | relative time (auto-updating)